repos/system_upgrade/common/actors/opensshconfigscanner/libraries/readopensshconfig.py

```diff
@@ -60,12 +61,35 @@ def parse_config(config, base_config=None, current_cfg_depth=0):
             # convert deprecated alias
             if value == "without-password":
                 value = "prohibit-password"
-            v = OpenSshPermitRootLogin(value=value, in_match=in_match)
+            try:
+                v = OpenSshPermitRootLogin(value=value, in_match=in_match)
+            except ModelViolationError:
+                valid_values = OpenSshPermitRootLogin.value.serialize()['choices']
+                raise StopActorExecutionError(
+                    'Invalid value for SSH configuration option',
```

Comment:
Thinking about that, I would update the error msg to something like this, so it's easier in future to search the solution for exact problem. We can expect that many users will not even read the full error details and they will blindly report just the title (yes, that's happening unfortunately). As we know clearly what's happening, I would go this way.

```suggestion
                    'Invalid SSH configuration: Invalid value for PermitRootLogin',
```